share/spice/sublime_packages/footer.handlebars

```diff
@@ -1,4 +1,11 @@
 <div class="tile__update__info one-line">
-	<i class="ddgsi ddgsi-arrow-down"></i>
-	{{unique_installs}} &middot; Updated {{last_modified}}
+    <div class="gw">
+        <span class="g   opt--left">
```

Comment:
I think you should be able to forego using the `gw` and `g` classes. Only the `opt--*` classes should be necessary to float these.